lib/src/track/remote_track_publication.dart

```diff
@@ -28,14 +30,14 @@ class RemoteTrackPublication extends TrackPublication {
   }
 
   @override
-  bool get subscribed {
+  bool get isSubscribed {
     if (_unsubscribed) {
       return false;
     }
-    return super.subscribed;
+    return super.isSubscribed;
   }
 
-  set subscribed(bool val) {
+  set isSubscribed(bool val) {
```

Comment:
```suggestion
setSubscribed(bool val) {
```

nit, but isXXXX usually indicates that it's a getter. feels a bit unusual to call `pub.isSubscribed = false`